geowebcache/core/src/main/java/org/geowebcache/GeoWebCacheDispatcher.java

```diff
@@ -352,9 +352,13 @@ private void handleServiceRequest(String serviceStr, HttpServletRequest request,
 
         // Check where this should be dispatched
         if (conv.reqHandler == Conveyor.RequestHandler.SERVICE) {
-            // A3 The service object takes it from here
-            service.handleRequest(conv);
-
+            try {
```

Comment:
Instead of doing this same thing inside the two branches if the `if` it would make more sense to wrap the entire `if`.